packages/tables/resources/views/columns/text-column.blade.php

```diff
@@ -211,6 +210,7 @@
                                         'fi-ta-text-item-label',
                                         'group-hover/item:underline group-focus-visible/item:underline' => $url,
                                         'line-clamp-[--line-clamp]' => $lineClamp,
+                                        'whitespace-normal' => $canWrap,
```

Comment:
Could you check if this change is required for the infolist text entry too?

Reply:
Found that infolist text entries are a little different, so won't handle that here.